### PR TITLE
Feature/update docker env and jupyter notebooks

### PR DESCRIPTION
**Pull request by MatheusClaudi**

### CODE REVIEW

File: docs/api/api_scripts/basic_flow.ipynb [hunk 1]
```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This notebook describes a basic flow of ml-git with api. There you will learn how to initialize an ml-git project, how to perform all necessary configuration steps and how to version a dataset. We will divide this quick howto into 3 main sections:\n",
+    "This notebook describes a basic execution flow of ml-git with its API. There, you will learn how to initialize an ML-Git project, how to perform all the necessary configuration steps and how to version a dataset. We will divide this quick howto into 3 main sections:\n",
```
Comment (andersonpablosilva) [inline docs/api/api_scripts/basic_flow.ipynb:14]:
```suggestion
    "This notebook describes a basic execution flow of ml-git with its API. There, you will learn how to initialize an ML-Git project, how to perform all the necessary configuration steps and how to version a dataset. We will divide this quick how-to into 3 main sections:\n",
```

**Review by andersonpablosilva — COMMENTED** [review]
Can we create these empties .md's dynamically? If we can navigate in the directory structure before the conversion, maybe we could execute the bash above to create these empty markdowns.

```bash
for jupyter_file in ./**/*.ipynb; do
    touch "${jupyter_file%.ipynb}.md"
done
```

File: docs/api/api_scripts/relationship_notebook/relationship_api_commands.ipynb [hunk 2]
```diff
@@ -17,6 +17,14 @@
     "\n",
     "In it, we'll show you how to use the set of commands provided. You can check the documentation for more information: [API documentation](https://github.com/HPInc/ml-git/blob/main/docs/api/README.md)\n",
     "\n",
+    "### Notebook prerequisites\n",
+    "\n",
+    "This notebook uses the GitHub API to access a reset and does Ml-Git operations on its contents, so before running this notebook, take the following steps:\n",
```
Comment (Vieira-Carlos) [inline docs/api/api_scripts/relationship_notebook/relationship_api_commands.ipynb:22]: What is a 'reset' here?

Comment (MatheusClaudi) [reply docs/api/api_scripts/relationship_notebook/relationship_api_commands.ipynb:22]: The notebooks do some operations on the current folder like clone a repository and init a repository, and operations in repository origin(minio), like commit. The reset allows notebooks in the same folder not to interfere with another by reseting current folder data.

Comment (Vieira-Carlos) [reply docs/api/api_scripts/relationship_notebook/relationship_api_commands.ipynb:22]:
```suggestion
    "This notebook uses the GitHub API to access a repository and perform Ml-Git operations on its contents (while also able to reset any changes), so before running this notebook, take the following steps:\n",
```
Does this conform with what the notebook does?

**Review by Vieira-Carlos — APPROVED** [review]

**Review by lucasncabral — APPROVED** [review]

**Review by andersonpablosilva — APPROVED** [review]

**Pull request merged**